project-addons/account_move_line_followup/models/credit_control_policy.py

```diff
@@ -111,8 +111,7 @@ def _get_unreconciled_move_lines(self):
                     ('move_id.state', '!=', 'draft'),
                     ('company_id', '=', self.company_id.id),
                     ('blocked', '!=', True),
-                    '|', ('invoice_id.state', '!=', 'paid'),
-                    ('invoice_id', '=', False),
+                    '|', ('debit', '>', 0), ('credit', '>', 0),
```

Comment:
¿No sería un and?

Reply:
Se podría substituir también el ('full_reconcile_id', '=', False), por ('reconciled', '=', False) como hice en los demás, que este domain no lo viera

Reply:
Lo del "and" creo que no, que es un "or", ya que los apuntes contables o tienen "Debe" (si es una factura de salida de cliente) o tienen "Haber" si proviene de una rectificativa, pero nunca coincide que sean los dos mayores que 0

Reply:
Lo otro ahora lo corrijo

Reply:
¿Pero el problema no era las facturas a 0? Si es este, tanto el debe como el haber es 0 en el apunte

Reply:
Sí, está bien, me lie yo.